### DISCUSSION TITLE
Keyboard shortcuts guide for contributors

### DISCUSSION THREAD

**Discussion by aiwrapper** [alan-turing-institute/sktime#1224] [Ideas]
I suggest making a list of the most useful shortcuts in VS Code and PyCharm to help contributors, especially new ones.

Alternatively, we could provide links to shortcuts and how they can be customised in editors.

This would potentially save precious time for many of us.

**Comment by aiwrapper** [comment]
VSCode's keyboard shortcuts: https://keycombiner.com/courses/master-vscode/

**Comment by fkiraly** [comment]
hm, this could be part of the developer set-up page?

**Comment by lovkush-shell** [comment]
I like the idea of encouraging the use of keyboard shortcuts, but there is already so much information for new contributors that this might be overwhelming. (Also worth noting that many new people do not use keyboard shortcuts outside of the IDE, eg using alt-tab to switch between windows.)
